app/build.gradle

```diff
@@ -237,7 +237,7 @@ dependencies {
     implementation Dependencies.Square.retrofitGson
     implementation Dependencies.Square.sqlBrite
     debugImplementation Dependencies.Square.haha
-    debugImplementation Dependencies.Square.leakCanaryDebug
+    debugImplementation Dependencies.Square.leakCanaryRel
     releaseImplementation Dependencies.Square.leakCanaryRel
```

Comment:
Do you have a link you can include in this PR that defines what `Dependencies.Square.leakCanaryRel` is? If it's effectively just a no-op then this is fine. If there is other release-related logic in this dependency then we don't want to include it in debug builds.

Reply:
leakCanaryRel is just a string representation of the dependency. Here's the code snippet from the Dependency.kt file from the project

Reply:
leakCanaryRel is just an object representation of the dependency. Here's the code snippet from the Dependency.kt file from the project
`const val leakCanary = "1.6.3"`
`const val leakCanaryRel = "com.squareup.leakcanary:leakcanary-android-no-op:${Versions.leakCanary}"`

Reply:
Ok, thanks. And do you have a link to what `leakcanary-android-no-op` dependency is? Seems obvious from the description, but I just want to make sure there aren't any things related to releases in here that shouldn't be in debug builds.

Reply:
Also - a change this like should have the commit prefix of `chore:`, because we're not fixing a bug.

Reply:
Ah, I think I missed that in the documentation. Thank you for letting me know.

Reply:
Hmm, so `leakcanary-android-no-op` is just a name of the dependency from what I understand from the documentation and I don't think it refers to anything in the code.

Reply:
Is there some documentation that explains what `leakcanary-android-no-op` is or when to use it?

Reply:
I couldn't find a direct link from the official LeakCanary website but however, I did find a few links from Stackoverflow that discuss the use of the no-op variant.

Reply:
Looks like the `no-op` library is from an older version of LeakCanary:
https://square.github.io/leakcanary/upgrading-to-leakcanary-2.0/#worth-noting

They stopped using it in v2. There is some discussion surrounding this in https://github.com/square/leakcanary/issues/979. Based on this discussion, I think it's fine to use this in debug builds.

Reply:
I couldn't find a direct link to the official LeakCanary website but however, I did find a few links from Stackoverflow that discuss the use of the no-op variant. 
https://stackoverflow.com/questions/43310444/should-i-remove-leak-canary-code-classes-for-release-build
https://stackoverflow.com/questions/33654503/how-to-use-leak-canary

Also, there's LeakCanary 2.4 that builds only for debug variant and there won’t be any code referencing LeakCanary in release builds.
https://square.github.io/leakcanary/upgrading-to-leakcanary-2.0/

Reply:
Yes, you're right. I guess I received your last message while I was typing my last comment but didn't get updated on my screen haha.